Source/Elasticurl/main.swift

```diff
@@ -238,20 +262,31 @@ struct Elasticurl {
             let semaphore = DispatchSemaphore(value: 0)
 
             var stream: HttpStream?
-            var connection: HttpClientConnection?
 
             let httpRequest: HttpRequest = HttpRequest(allocator: allocator)
-            httpRequest.method = "GET"
-            httpRequest.path = "/"
+            httpRequest.method = context.verb
+            let path = context.url.path == "" ? "/" : context.url.path
+
+            httpRequest.path = path
 
             let headers = HttpHeaders(allocator: allocator)
-            if headers.add(name: "Host", value: context.url),
-                headers.add(name: "User-Agent", value: "Elasticurl"),
-                headers.add(name: "Accept", value: "*/*") {
+            if headers.add(name: "Host", value: host),
+               headers.add(name: "User-Agent", value: "Elasticurl"),
```

Comment:
Probably want to add swift, maybe a version?

Reply:
added